core/src/View/MDSpan/Kokkos_MDSpan_Accessor.hpp

```diff
@@ -81,8 +83,7 @@ struct SpaceAwareAccessor {
   }
 
   KOKKOS_FUNCTION
-  constexpr data_handle_type offset(data_handle_type p, size_t i) const
-      noexcept {
+  constexpr auto offset(data_handle_type p, size_t i) const noexcept {
```

Comment:
The return type should really be `typename offset_policy​::​data_handle_type` (according to https://eel.is/c++draft/mdspan.accessor.reqmts#9), right?

Reply:
jup but that is the same as nested_acc.offset return type

Reply:
I would prefer spelling out the return type here instead of using `auto`.

Reply:
done